src/components/ManageTeamHealth.vue

```diff
@@ -42,35 +61,81 @@ export default {
     },
 
     reset() {
-      this.$emit('reset');
+      // TODO: Reset i
       this.hasChanged = false;
     },
 
     changeStatus(team, key) {
       this.hasChanged = false;
-      if (team.criteria_values[key] === 3) {
-          team.criteria_values[key] = 1;
+      if (team.criteria_values[key] === STATUS.GREEN) {
+          team.criteria_values[key] = STATUS.RED;
       }
       else {
           team.criteria_values[key]++;
       }
       this.hasChanged = true;
     },
+
+    /**
+     * Compares a team's criteria value with the previous sprint value, if it exists, 
+     * to determine whether there was an increase or decrease. 
+     * @returns -1 for decrease, 0 for no change, 1 for increase
```

Comment:
Do you want to put the params in? Don't have to just thought seen as though you added a JSDoc comment you might want to :) as in the past the params have been what messed me up (I think I am forever scarred by it lol).
I will out them in too going forward.